### PR TITLE
Add Symfony 5 support

### PR DESCRIPTION
**Pull request by wizu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
         ]
     },
     "require": {
-        "php": ">=7.1",
-        "sebastian/comparator": "^1.2 | ^2.0 | ^2.1 | ^3.0",
-        "symfony/property-access": "^2.7 | ^3.0 | ^4.0"
+        "php": ">=7.3",
```
Comment (jdreesen) [inline composer.json:26]: This should not be necessary.

Comment (pierredup) [reply composer.json:26]: Agree, the minimum php version should remain the same

Comment (wizu) [reply composer.json]: Yeah, right.

**Comment by pierredup** [comment]
Thank you @wizu

**Pull request merged**